sympy/core/relational.py

```diff
@@ -371,6 +384,14 @@ def _eval_trigsimp(self, **opts):
         from sympy.simplify import trigsimp
         return self.func(trigsimp(self.lhs, **opts), trigsimp(self.rhs, **opts))
 
+    def expand(self, **kwargs):
+        args = (arg.expand(**kwargs) for arg in self.args)
+        return self.func(*args)
+
+    def integrate(self, *args, **kwargs):
+        """See the integrate function in sympy.integrals"""
+        from sympy.integrals import integrate
+        return integrate(self, *args, **kwargs)
```

Comment:
I don't think that integration of relationals has been implemented.

Reply:
This method is being depended upon somewhere. I'll have a look...

Reply:
There is a test here:
https://github.com/sympy/sympy/blob/121ccd76879eabcb9567cb6f88d976f761f171d1/sympy/integrals/tests/test_integrals.py#L1506-L1509

That comes from #15530

Reply:
It seems that there is a special construction for `Integral(Eq)` that transforms it to `Eq(Integral, Integral)` before `doit()` is applied. For other relations there will be integration errors. Maybe we could add an error message for those here.

Reply:
Would it make sense to define this as a method of `Equality` alone?

Reply:
I think it would be better to remove/deprecate both `Integral(Eq)` and `Eq.integrate`. They aren't valid for indefinite integrals anyway (there should be an integration constant).

Reply:
Maybe we shouldn't deprecate both. Someone may find it convenient to use `Eq.integrate` (though I personally probably would not). But I think that we could deprecate `Integral(Eq)` by adding the message to the transformation code in `_common_new()`. Then `Eq.integrate` could do the transformation and return `Eq(integrate(), integrate())`. It could also check the limits and accept only definite integrals.